### PR TITLE
LN Pages refactoring

### LINKED ISSUE #3259: [css modules] Coordination issue.
This issue will be used to coordinate `.less` files migration work - original issue: #2439.

Following the list of all `.less` files existed before the migration - checked checkbox indicates that `.less` was migrated:

 - [x] AccountRow.less
 - [x] AccountPage.less
 - [x] AgendaCard.less
 - [x] AgendaOverview.less
 - [x] Balance.less
 - [ ] ConnectPage.less
 - [x] Chart.less
 - [x] CreateWalletForm.less
 - [x] Documentation.less
 - [x] ExportPage.less
 - [x] EyeFilterMenu.less
 - [x] Fonts.less
 - [x] GetStarted.less
 - [x] Governance.less
 - [x] Header.less
 - [x] Heatmap.less
 - [x] Help.less
 - [x] HistoryPage.less
 - [x] HomePage.less
 - [x] Icons.less
 - [x] Input.less
 - [ ] LN.less
 - [x] LanguageSelect.less
 - [x] Layout.less
 - [x] Loading.less
 - [x] Logs.less
 - [x] MiscComponents.less
 - [x] Modals.less
 - [x] MyTickets.less
 - [x] Paginator.less
 - [x] PurchaseTickets.less
 - [x] ReceivePage.less
 - [x] ReleaseNotes.less
 - [x] SecurityCenterMssagePage.less
 - [x] SendPage.less
 - [x] Settings.less
 - [x] SideBar.less
 - [x] SimpleLoading.less
 - [x] Snackbar.less
 - [x] StakePool.less
 - [x] Stakey.less
 - [x] TextToggle.less
 - [x] Themes.less
 - [x] Tooltip.less
 - [x] Trezor.less
 - [x] Tutorial.less
 - [x] TxDetails.less
 - [x] TxHistory.less
 - [x] main.less

### LINKED ISSUE #3260: class => functional components & hooks migration coordination.
All unchecked class components in the following list weren't migrated yet from class components to a functional component
with hooks & connectors :

- [x] `components/ErrorScreen`
- [x] `components/shared/Balance`
- [x] `components/buttons/*`
- [x] `components/Header`
- [x] `components/Paginator`
- [x] `components/inputs/*`
- [x] `components/inputs/AccountSelector.js`
- [x] `test/unit/components/inputs/DcrInput.spec.js`
- [x] `components/layout/*`
- [x] `components/views/TicketsPage/PurchaseTab/StakeInfo/StakeInfo.jsx` 
- [x] `components/modals/trezor/*`
- [x] `components/views/TrezorPage/*`
- [x] `components/indicators/LoaderBarBottom.js`
- [ ] `components/views/LNPage/*` - **WIP**
- [x] `containers/App.js`
- [x] `components/views/TicketsPage/PurchaseTab/LEGACY_PurchasePage/LEGACY_PurchaseTickets/index.js`
- [x] `components/views/TicketsPage/PurchaseTab/LEGACY_PurchasePage/LEGACY_TicketAutoBuyer/index.j`

Original issue: #2438

### PR DESCRIPTION
**Pull request by guilhermemntt**
This PR implements the approach proposed at #2438 and #2439 to app/components/views/LNPage/* components, this way finishing the commitment made at #3260. Closes #3260 and closes #3259.

Before:

image

After:

image

### CODE REVIEW

File: app/components/views/LNPage/PaymentsTab/PaymentsTab.jsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+      <Subtitle
+        title={<T id="ln.paymentsTab.sendPayment" m="Send Payment" />}
+      />
+      <div className={styles.lnSendPayment}>
+        <div className="payreq">
+          <T id="ln.paymentsTab.payReq" m="Payment Request" />
+          <TextInput value={payRequest} onChange={onPayRequestChanged} />
+        </div>
+        {!!decodingError && (
+          <div className={styles.decodingError}>{"" + decodingError}</div>
```
Comment (amassarwi) [inline app/components/views/LNPage/PaymentsTab/PaymentsTab.jsx:56]:
```suggestion
          <div className={styles.decodingError}>{decodingError || ""}</div>
```

File: app/components/views/LNPage/PaymentsTab/hooks.js [hunk 2]
```diff
@@ -1,20 +1,102 @@
...
+      })
+      .catch((error) => {
+        setDecodedPayRequest(null);
+        setDecodingError(error);
+      });
+  }, [payRequest, decodePayRequest, checkExpired, setTimeout]);
+
+
+  const onPayRequestChanged = (e) => {
+    setPayRequest(("" + e.target.value).trim());
```
Comment (amassarwi) [inline app/components/views/LNPage/PaymentsTab/hooks.js:67]:
```suggestion
    setPayRequest((e.target.value || "").trim());
```

File: app/components/views/LNPage/WalletTab/WalletTab.module.css [hunk 3]
```diff
@@ -72,20 +72,11 @@
...
-  word-break: break-all;
-  padding-right: 40px;
-  padding-bottom: 8px;
-}
-
 .backupInfoBtn {
   margin-right: 2px;
 }
 
+/* BalanceHeader */
```
Comment (amassarwi) [inline app/components/views/LNPage/WalletTab/WalletTab.module.css:79]: I would suggest creating `BalanceHeader.module.css` where there classes are used

File: app/containers/Wallet.jsx [hunk 4]
```diff
@@ -15,7 +15,7 @@ import TutorialsPage from "components/views/TutorialsPage/TutorialsPage";
 import GovernancePage from "components/views/GovernancePage/GovernancePage";
 import ProposalDetailsPage from "components/views/ProposalDetailsPage/ProposalDetailsPage";
 import TrezorPage from "components/views/TrezorPage";
-import LNPage from "components/views/LNPage";
+import LNPage from "components/views/LNPage/LNPage";
```
Comment (amassarwi) [inline app/containers/Wallet.jsx:18]: create `index.js` file to avoid this extra verbose.

**Review by amassarwi — CHANGES_REQUESTED** [review]
@guilhermemntt Good job on this one mate, code looks neat, left few suggestions.

**Comment by amassarwi** [comment]
Probably the LN guys want to try this out - @matheusd  @fguisso @vctt94  

I'll be setting testnet LN wallet now to give it a try as well.

**Review by matheusd — COMMENTED** [review]
Functionally, seems to be working fine

**Comment by amassarwi** [comment]
Please add Closes https://github.com/decred/decrediton/issues/3260 & Closes https://github.com/decred/decrediton/issues/3259 to the PR's description, as this commit refactors the last piece to functional approach with css modules.

I will follow up with 1-2 cleanup PRs tonight which will finally close #2438 & #2439. Historic moment 🎉.

**Comment by guilhermemntt** [comment]
Rebased @amassarwi !

**Review by alexlyp — APPROVED** [review]

**Pull request merged**